src/Redux/books/bookSlice.js

```diff
@@ -1,20 +1,46 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  Booklist: [],
+  Booklist: [
+    {
+      item_id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+      Percentage: '10%',
+      Chapter: '17',
+    },
+    {
+      item_id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+      Percentage: '0%',
+      Chapter: 'Introduction',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+      Percentage: '90%',
+      Chapter: 'The last',
+    },
+  ],
 };
 
 export const booklists = createSlice({
   name: 'bookLists',
   initialState,
   reducers: {
     addBook: (state, action) => {
-      state.Booklist.pop(action.payload);
+      state.Booklist.push(action.payload);
```

Comment:
- [Optional] Great Job for adding the add book reducer, I think it would be better if you used nonmutable array methods to improve the performance 👍
`state.books = [...state.books, action.payload]`